form.js

```diff
@@ -1,25 +1,25 @@
-// Load form data from local storage on page load
-window.addEventListener('DOMContentLoaded', () => {
-  const formData = localStorage.getItem('form_data');
-  if (formData) {
-    const {
-      name, firstName, lastName, email, textArea,
-    } = JSON.parse(formData);
-    document.getElementById('name').value = name || '';
-    document.getElementById('first_name').value = firstName || '';
-    document.getElementById('last_name').value = lastName || '';
-    document.getElementById('email').value = email || '';
-    document.getElementById('text_area').value = textArea || '';
-  }
+// Get form element and submit button
+const form = document.getElementById('form');
+
+// Add event listener to the form submission
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const nameInput = document.getElementById('name');
+  const emailInput = document.getElementById('email');
+  const textArea = document.getElementById('text_area');
+  localStorage.setItem('name', nameInput.value);
+  localStorage.setItem('email', emailInput.value);
+  localStorage.setItem('text', textArea.value);
```

Comment:
[OPTIONAL] It wouldn't be a bad idea to set data to local Storage once and for all as an object like this 
```
  localStorage.setItem('inputs', {
   name: ,
   email: ,
   text: 
  })
```